command/connect/envoy/testdata/defaults.golden

```diff
@@ -19,27 +19,69 @@
         "connect_timeout": "1s",
         "type": "STATIC",
         "http2_protocol_options": {},
-        "hosts": [{
+        "hosts": [
+          {
             "socket_address": {
               "address": "127.0.0.1",
               "port_value": 8502
             }
-          }]
+          }
+        ]
       }
     ]
   },
   "stats_config": {
-			"stats_tags": [
-				{
-			"tag_name": "local_cluster",
-			"fixed_value": "test-proxy"
-		}
-			],
-			"use_all_default_tags": true
-		},
+    "stats_tags": [
+      {
+        "regex": "^cluster\\.((?:([^.]*)~)?(?:[^.]*\\.)?[^.]+\\.[^.]+\\.[^.]+\\.[^.]+\\.[^.]+\\.consul\\.)",
+        "tag_name": "consul.custom_hash"
+      },
+      {
+        "regex": "^cluster\\.((?:[^.]*~)?(?:([^.]*)\\.)?[^.]+\\.[^.]+\\.[^.]+\\.[^.]+\\.[^.]+\\.consul\\.)",
+        "tag_name": "consul.service_subset"
+      },
+      {
+        "regex": "^cluster\\.((?:[^.]*~)?(?:[^.]*\\.)?([^.]+)\\.[^.]+\\.[^.]+\\.[^.]+\\.[^.]+\\.consul\\.)",
+        "tag_name": "consul.service"
+      },
+      {
+        "regex": "^cluster\\.((?:[^.]*~)?(?:[^.]*\\.)?[^.]+\\.([^.]+)\\.[^.]+\\.[^.]+\\.[^.]+\\.consul\\.)",
+        "tag_name": "consul.namespace"
+      },
+      {
+        "regex": "^cluster\\.((?:[^.]*~)?(?:[^.]*\\.)?[^.]+\\.[^.]+\\.([^.]+)\\.[^.]+\\.[^.]+\\.consul\\.)",
+        "tag_name": "consul.datacenter"
+      },
+      {
+        "regex": "^cluster\\.((?:[^.]*~)?(?:[^.]*\\.)?[^.]+\\.[^.]+\\.[^.]+\\.([^.]+)\\.[^.]+\\.consul\\.)",
+        "tag_name": "consul.visibility"
```

Comment:
Names are hard! I don't feel crazy strongly about this but the internal/external thing seems to me less about visibility (which implies that internal stuff is somehow more secure or hidden) and more about whether it's a "last mile" address after discovery chain has already resolved and appropriate stuff is selected vs a hypothetical other sort of address that should already be subject to routing.

As such a name like `routing_type` feels better although it's probably not meaningful to the user. Thoughts?

Reply:
I would love any name other than the one I picked. `routing_type` sounds like a fine alternative.